amadeus-integration-test/FlightOrderManagementTest.cs

```diff
@@ -25,12 +24,9 @@ public void Get_FlightOrderManagement()
+            string body = "{\"data\":{\"type\":\"flight-order\",\"flightOffers\":[{\"type\":\"flight-offer\",\"id\":\"1\",\"source\":\"GDS\",\"instantTicketingRequired\":false,\"nonHomogeneous\":false,\"oneWay\":false,\"lastTicketingDate\":\"2022-01-21\",\"numberOfBookableSeats\":7,\"itineraries\":[{\"duration\":\"PT5H49M\",\"segments\":[{\"departure\":{\"iataCode\":\"JFK\",\"terminal\":\"4\",\"at\":\"2022-02-01T10:31:00\"},\"arrival\":{\"iataCode\":\"LAX\",\"terminal\":\"2\",\"at\":\"2022-02-01T13:20:00\"},\"carrierCode\":\"DL\",\"number\":\"351\",\"aircraft\":{\"code\":\"764\"},\"operating\":{\"carrierCode\":\"DL\"},\"duration\":\"PT5H49M\",\"id\":\"30\",\"numberOfStops\":0,\"blacklistedInEU\":false}]}],\"price\":{\"currency\":\"EUR\",\"total\":\"122.53\",\"base\":\"101.00\",\"fees\":[{\"amount\":\"0.00\",\"type\":\"SUPPLIER\"},{\"amount\":\"0.00\",\"type\":\"TICKETING\"}],\"grandTotal\":\"122.53\"},\"pricingOptions\":{\"fareType\":[\"PUBLISHED\"],\"includedCheckedBagsOnly\":false},\"validatingAirlineCodes\":[\"DL\"],\"travelerPricings\":[{\"travelerId\":\"1\",\"fareOption\":\"STANDARD\",\"travelerType\":\"ADULT\",\"price\":{\"currency\":\"EUR\",\"total\":\"122.53\",\"base\":\"101.00\"},\"fareDetailsBySegment\":[{\"segmentId\":\"30\",\"cabin\":\"ECONOMY\",\"fareBasis\":\"TA7NA0BQ\",\"brandedFare\":\"BASICECON\",\"class\":\"E\",\"includedCheckedBags\":{\"quantity\":0}}]}]}],\"travelers\":[{\"id\":\"1\",\"dateOfBirth\":\"1987-07-22\",\"name\":{\"firstName\":\"amir\",\"lastName\":\"taheri\"},\"gender\":\"MALE\",\"documents\":[{\"documentType\":\"PASSPORT\",\"number\":\"J12345678\",\"expiryDate\":\"2024-02-22\",\"issuanceCountry\":\"TR\",\"nationality\":\"TR\",\"holder\":true}]}],\"contacts\":[{\"companyName\":\"INCREIBLEVIAJES\",\"purpose\":\"STANDARD\",\"phones\":[{\"deviceType\":\"LANDLINE\",\"countryCallingCode\":\"34\",\"number\":\"480080071\"},{\"deviceType\":\"MOBILE\",\"countryCallingCode\":\"33\",\"number\":\"480080072\"}],\"emailAddress\":\"support@increibleviajes.es\"}]}}";
             
-            FlightOrderCreateQuery response = flightOrderManagement.postFlightOrderManagement(body);
+            FlightOrderCreateQuery response = amadeus.referenceData.booking.flightOrderManagement.postFlightOrderManagement(body);
```

Comment:
This one doesn't belong under referenceData, since the URL is /booking/flight-orders